src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -176,16 +149,24 @@ public static class TargetFrameworkUtil
         public static ImmutableArray<MetadataReference> MinimalReferences => ImmutableArray.Create(TestBase.MinCorlibRef);
         public static ImmutableArray<MetadataReference> MinimalAsyncReferences => ImmutableArray.Create(TestBase.MinAsyncCorlibRef);
 
+#if DEBUG
+
+        static TargetFrameworkUtil()
+        {
+            Debug.Assert(GetReferences(TargetFramework.NetCoreApp).SequenceEqual(NetCoreApp.References));
```

Comment:
This will ensure going forward that we keep both definitions of `NetCoreApp` in sync.